vendor-extra/LiberoPageBundle/src/EventListener/BuildView/ContentHeaderAffiliationDuplicateListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\TemplateView;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewBuildingListener;
+use function array_unique;
+use function array_values;
+use function Libero\ViewsBundle\array_has_key;
+use const SORT_REGULAR;
+
+final class ContentHeaderAffiliationDuplicateListener
+{
+    use ViewBuildingListener;
+
+    protected function handle(Element $object, TemplateView $view) : View
+    {
+        $affiliations = $view->getArgument('affiliations');
+
+        $affiliations['items'] = array_values(
+            array_unique(
+                $view->getArgument('affiliations')['items'] ?? [],
+                SORT_REGULAR
```

Comment:
This is a bit too simplistic (a string and a `StringTemplate` should be considered the same if their text is the same), but can be improved later.

Reply:
> This is a bit too simplistic (a string and a StringTemplate should be considered the same if their text is the same), but can be improved later.

Would it be appropriate to add a comment in the code or open an issue and link it here?